dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -32,13 +32,43 @@
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.ParseException;
 
+/**
+ * Option for {@link StaticLinkageChecker}. To construct an input class path, the checker requires
+ * either one of the following types of input:
+ *
+ * <ul>
+ *   <li>{@code bomCoordinate}: a Maven coordinate for a BOM
+ *   <li>{@code mavenCoordinates}: list of Maven artifact coordinates
+ *   <li>{@code jarFileList}: list of jar files in the filesystem
+ * </ul>
+ *
+ * @see <a
+ *     href="https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/dependencies#input">Static
+ *     Linkage Checker: Input</a>
+ */
 @AutoValue
 abstract class StaticLinkageCheckOption {
   // TODO(suztomo): Add option to specify entry point classes
 
+  /**
+   * Returns a Maven coordinate for a BOM if specified; otherwise null.
+   */
   @Nullable abstract String getBomCoordinate();
+
+  /**
+   * Returns Maven coordinates if specified; otherwise an empty list.
```

Comment:
but maven coordinates for what? Why is this field here, especially compared to the BOM coordinates? For example (not sure if this is correct)

returns the Maven coordinates of the artifact being scanned for static linkage errors

or 

returns the Maven coordinates of the artifact that has a.static linkage error

These are two *different* things and from the code alone I can't tell which is intended.